src/commands/assume-role-with-web-identity.yml

```diff
@@ -0,0 +1,36 @@
+description: |
+  Generate a shortlived AWS_ACCESS_KEY_ID, AWS_ACCESS_KEY_ID and AWS_SESSION_TOKEN using a the $CIRCLE_OIDC_TOKEN.
+  A properly configured web identity based ARN is also required for configuration.
+  Use these keys and create a profile the aws-cli/setup command
+
```

Comment:
We may need to consider how to send users to more information for help/docs. We may want to link to the OIDC docs page on CircleCI (I know it wont link, and thats a bummer, but probably worth it). We might write a blurb in those docs later specifically about AWS too.

Reply:
Here's the link to our docs: `https://circleci.com/docs/2.0/openid-connect-tokens/`
I feel like this blog post is worthwhile too. What do you think? - `https://circleci.com/blog/openid-connect-identity-tokens/`

Reply:
Lets do just the docs for here. The blog posts feels less permanent.